src/lib/uTop.ml

```diff
@@ -15,6 +15,12 @@ open LTerm_text
 open LTerm_geom
 open LTerm_style
 
+module M = Directories.Project_dirs (struct
```

Comment:
`M` is not a good name - because what would it stand for? I assume this module provides `M.cache_dir`. A better name would be `D`, `Dirs`, or `Project`.

Reply:
What about `Project_dirs` ?

Reply:
That would work as well.